content/concepts/configuration.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Configuration
+---
+
+## Overview 
+You may have noticed that few webpack configurations look exactly alike. This is because **webpack's configuration file is a JavaScript file that exports an object.** This object, is then parsed by webpack based upon its defined properties.
+
+### Examples
+The following examples below describe how webpacks configuration object can be both expressive and configurable because _it is code_: 
+
+#### The simplest of examples
+**webpack.config.js**
+```javascript 
+module.exports = {
+  entry: './foo.js',
+  output: {
+    path: 'dist',
+    filename: 'foo.bundle.js'
+  }
+}
+
+```
+
+#### Generating multiple targets and configs and exporting the entire array.
+**webpack.config.js**
+```javascript
+var path = require('path');
+var webpack = require('webpack');
+var webpackMerge = require('webpack-merge');
+
+var baseConfig = {  
+  target: 'async-node',
+  entry: {
+    entry: './entry.js'
+  },
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, './dist')
+  },
+  plugins: [
+    new webpack.optimize.CommonsChunkPlugin({
+      name: 'inline',
+      filename: 'inline.js',
+      minChunks: Infinity
+    }),
+    new webpack.optimize.AggressiveSplittingPlugin({
+        minSize: 5000,
+        maxSize: 10000
+    }),
+  ]
+};
+
+let targets = ['web', 'webworker', 'node', 'async-node', 'node-webkit', 'electron-main'].map((target) => {
```

Comment:
FWIW, if it is a const, the user wouldn't be able to update it later :smirk: which we've all seen before in the same file. (Also I'm only a const fan for real const and imports but thats opinionated bias).